tests/test_mmar_download.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.apps import download_mmar, load_from_mmar
+from monai.apps.mmars import MODEL_DESC
+from tests.utils import skip_if_quick
+
+TEST_CASES = [["clara_pt_prostate_mri_segmentation_1"], ["clara_pt_covid19_ct_lesion_segmentation_1"]]
+TEST_EXTRACT_CASES = [
+    (
+        {
+            "item": "clara_pt_prostate_mri_segmentation_1",
+            "map_location": "cuda" if torch.cuda.is_available() else "cpu",
+        },
+        "UNet",
+        np.array(
+            [
+                [[-0.0838, 0.0116, -0.0861], [-0.0792, 0.2216, -0.0301], [-0.0379, 0.0006, -0.0399]],
```

Comment:
I am not sure whether we have fixed versions of MMARs, otherwise the test data can match once updated.

Reply:
the version info is encoded in the URL, I think this is unlikely to be changed

Reply:
ok, sounds good.